article/app/views/package.scala

```diff
@@ -36,7 +38,7 @@ object MainCleaner {
 object BodyCleaner {
   def apply(article: Article, html: String, amp: Boolean)(implicit request: RequestHeader) = {
     implicit val edition = Edition(request)
-    val cleaners = List(
+    val cleaners = ListBuffer(
```

Comment:
Making this list mutable. Since it is only a local variable it shouldn't be a issue.